temporal/t.rast.aggregate/testsuite/test_aggregation_absolute.py

```diff
@@ -236,6 +236,43 @@ def test_aggregation_3months(self):
         maps = "b_101" + os.linesep
         self.assertEqual(maps, lister.outputs.stdout)
 
+    def test_weighted_aggregation(self):
+        """Weighted aggregation to 3 weeks"""
+        self.assertModule(
+            "t.rast.aggregate",
+            input="A",
+            output="B",
+            basename="b",
+            granularity="3 weeks",
+            method="average",
+            sampling=["related"],
+            file_limit=0,
+            suffix="num%03",
+            flags="w",
+        )
+        # assert that there are 5 rasters
+        t_rast_list = SimpleModule("t.rast.list", input="B", flags="u").run()
+        rasters = [
+            item
+            for item in t_rast_list.outputs.stdout.split(os.linesep)
+            if (len(item) > 0 and item.startswith("b_"))
```

Comment:
startswith will be false when len(item) == 0, so no need to check len.

Reply:
```suggestion
            if item.startswith("b_")
```